assembl/static2/js/app/graphql/IdeaWithPosts.graphql

```diff
@@ -8,7 +8,7 @@ query IdeaWithPosts($lang: String!, $id: ID!) {
       posts(first: 20) {
         edges {
           node {
-            ... on Post {
+            ... on AssemblPost {
```

Comment:
I tried to use an union or interface type instead (defined in graphql/schema.py) but I could not get it to work... Any idea @vincentfretin @maparent @cedricmessiant ?